scripts/pychop/__init__.py

```diff
@@ -12,15 +12,19 @@
 
 Usage:
 
-from mantidqtinterfaces.PyChop import PyChop2
+from pychop.Instruments import Instrument
 
-merlin = PyChop2('merlin','s',250)
+merlin = Instrument('merlin','s',250)
 merlin.setEi(25)
 resolution = merlin.getResolution()
 
-resolution, flux = PyChop2.calculate(inst='maps', chtyp='a', freq=500, ei=600, etrans=range(0,550,50))
-
-PyChop2.showGUI()
+resolution, flux = Instrument.calculate(inst='maps', chtyp='a', freq=500, ei=600, etrans=range(0,550,50))
 """
 
-from .Instruments import Instrument as PyChop2  # noqa: F401
+
+def PyChop2(instrument, chopper=None, freq=None):
+    import warnings
+    warnings.warn("Deprecation Warning: Importing 'PyChop2' from the 'mantidqtinterfaces.PyChop' module is deprecated."
+                  "Please import 'Instrument' from the 'pychop.Instruments' module instead.", DeprecationWarning)
+    from pychop.Instruments import Instrument
+    return Instrument(instrument, chopper, freq)
```

Comment:
Instead of specific positional arguments can you just pass `*args, **kwargs` - e.g.

```
def PyChop2(*args, **kwargs):
    ...
    return Instrument(*args, **kwargs)
```

This will mirror the current `PyChop2` syntax which is not just restricted to the three positional arguments...